README.rst

```diff
@@ -265,14 +265,13 @@ You can install it by doing the following :
 
 .. code-block:: bash
 
+        $ python3 -m venv .venv
```

Comment:
I'm not sure venv management should be covered here. There are tons of ways of to manage a venv, and forcing one in a tutorial ends up badly in my experience. Just a mention of a venv should be enough, as is before.

Reply:
This is Python's built-in way of creating and enabling a venv.  Users are free to choose alternative methods but I prefer explicit steps here to document what we know works and avoid issues being opened around implicit steps.

Reply:
Opinionated tooling would have been a suggestion to use `uv`. 😄